### ISSUE TITLE
Importing Plone 4.3- site in Plone 5 fails: No module named CMFDefault.DiscussionItem

### ISSUE THREAD

**Issue by schorsch2** [plone/Products.CMFPlone#800]
trying out an new installed plone 5b3 with importing my plone 4.3- site gives me this error:
Error Type: ImportError
Error Value: No module named CMFDefault.DiscussionItem
How can this problem be fixed?

**Comment by vangheem** [comment]
Please provide full traceback.

What add-ons do you have installed?

**Comment by schorsch2** [comment]
2015-08-05T16:23:40 ERROR Zope.SiteErrorLog 1438784620.610.164539220295 http://linux:48080/manage_importObject
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module OFS.ObjectManager, line 627, in manage_importObject
  Module OFS.ObjectManager, line 645, in _importObjectFromFile
  Module ZODB.ExportImport, line 92, in importFile
  Module transaction._transaction, line 260, in savepoint
  Module transaction._transaction, line 257, in savepoint
  Module transaction._transaction, line 690, in __init__
  Module ZODB.Connection, line 1123, in savepoint
  Module ZODB.Connection, line 587, in _commit
  Module ZODB.ExportImport, line 176, in _importDuringCommit
ImportError: No module named CMFDefault.DiscussionItem

no add-ons are installed - in both Plone5 and 4.3

**Comment by vangheem** [comment]
Was this originally a 4.3 site? Looks like there are some very old comments on there that aren't migrated--maybe never got migrated to plone.app.discussion. 

@tisto ideas?

**Comment by tisto** [comment]
@schorsch2 what does "importing" mean? ZEXP import? As @vangheem pointed out it seems that you have some very old comments in your ZODB that should have been migrated in the 4.x migrations.

**Comment by schorsch2** [comment]
true, it means ZEXP import
true, this site started by 2010 (maybe earlier)
How do I resolve these comments? I do not need discussion on this site.

**Comment by vangheem** [comment]
ZEXP export/import is only meant to be used on sites that were setup with same plone version and add-ons or it can lead to problems.

You'll need to upgrade your Plone site or use something like https://github.com/collective/collective.jsonify to export and then import.

Closing as this is actually not an upgrade problem but a mis-use of zexp exports.

**Comment by vangheem** [comment]
Also, if you're looking for help on upgrading, using stackoverflow instead of creating an issue here is probably a better place.

**Comment by fgrcon** [comment]
In case someone runs into this issue again:
As far i can say  I have followed all recommended  upgrade procedures but got the same problem.

 I solved it that way:
Add **Products.CMFDefault** to buildout (was obviously removed from Plone 4 to 5?).
Import of zexps run smoothly.
Afterwards I removed : 
- Products.CMFCore.interfaces.IDiscussable
- Products.CMFCore.interfaces.IDiscussionResponse
- Products.CMFCore.interfaces.IDiscussionTool
using  **wildcard.fixpersistentutilities** and  also removed Products.CMFDefault from buildout again.-
